interpreter/runtimecost.go

```diff
@@ -144,9 +146,25 @@ func NewCostTracker(estimator ActualCostEstimator, opts ...CostTrackerOption) (*
 	return tracker, nil
 }
 
+// FunctionCostTracker binds a function overload to a runtime FunctionTracker implementation.
+//
+// FunctionCostTracker instances augment or override ActualCostEstimator decisions, allowing for  versioned and/or
+// optional cost tracking changes.
+func FunctionCostTracker(function, overloadID string, fnTracker FunctionTracker) CostTrackerOption {
+	return func(tracker *CostTracker) error {
+		functionKey := fmt.Sprintf("%s|%s", function, overloadID)
+		tracker.functionTrackers[functionKey] = fnTracker
+		return nil
```

Comment:
return nil directly here? Does that mean there will never be error here?

Reply:
That's correct, not from this particular `CostTrackerOption`